server-embedded/src/main/java/org/exoplatform/chat/services/mongodb/UserMongoDataStorage.java

```diff
@@ -465,6 +465,8 @@ public void setSpaces(String user, List<SpaceBean> spaces)
       doc.put("_id", user);
       doc.put("user", user);
       doc.put("spaces", spaceIds);
+      doc.put("isEnabled", Boolean.toString(true));
+      doc.put("isDeleted", Boolean.toString(false));
```

Comment:
```suggestion
      doc.put("isEnabled", "true");
      doc.put("isDeleted", "false");
```

Reply:
I did it on purpose to make sure that the person reading the code will understand that we need a String instead of a boolean.